src/libAtomVM/nifs.c

```diff
@@ -5842,6 +5851,50 @@ static term nif_erlang_lists_subtract(Context *ctx, int argc, term argv[])
     return result;
 }
 
+
+static term nif_file_get_cwd_0(Context *ctx, int argc, term argv[])
+{
+    UNUSED(argc)
+    UNUSED(argv)
+
+    char cwd[PATH_MAX];
+    if (IS_NULL_PTR(getcwd(cwd, PATH_MAX))) {
```

Comment:
`PATH_MAX` is not defined on STM32 and RPi Pico, also more broadly `getcwd()` is POSIX but not standard C, so I don't expect it to be available on all platforms.
We might use some kind of `HAVE_CWD` define to check if a platform has it or not, as we are already doing with `open()` and other POSIX functions.
Question: should we move this to `posix_nifs.c`? CC @pguyot

Reply:
Indeed, if we test for HAVE_GETCWD because it's posix and not C, I'd say it makes sense to move it to `posix_nifs.c`.